src/main/java/nextstep/step2/domain/Lottos.java

```diff
@@ -0,0 +1,45 @@
+package nextstep.step2.domain;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Lottos {
+	private List<Lotto> lottos;
+
+	public Lottos(List<Lotto> lottos) {
+		this.lottos = lottos;
+	}
+
+	public List<Lotto> getLottos() {
+		return lottos;
+	}
+
+	public static List<Lotto> purchaseManualLottos(String manualLottos) {
+		String[] lottoNumbers = manualLottos.split(":");
+		List<Lotto> lottos = new ArrayList<>(lottoNumbers.length);
+		for(String numbers : lottoNumbers) {
+			lottos.add(Lotto.of(numbers));
+		}
+		return lottos;
+	}
+
+	public static List<Lotto> purchaseAutoLottos(int lottoCount) {
+		List<Lotto> lottos = new ArrayList<>(lottoCount);
+		for(int i = 0; i < lottoCount; i++) {
+			lottos.add(LottoAutoGenerator.getAutoNumbers());
+		}
+		return lottos;
+	}
+
+	public static Lottos mergeLottos(String lottoNumbers, int lottoCount) {
+		List<Lotto> manualLottos = purchaseManualLottos(lottoNumbers);
+		List<Lotto> autoLottos = purchaseAutoLottos(lottoCount);
+
+		return Stream.of(manualLottos, autoLottos)
+				.flatMap(Collection::stream)
+				.collect(Collectors.collectingAndThen(Collectors.toList(), Lottos::new));
+	}
```

Comment:
```suggestion
	public static Stream<Lotto> purchaseManualLottos(String manualLottos) {
		return Stream.of(manualLottos.split(":"))
		        .map(Lotto::of);
	}

	public static Stream<Lotto> purchaseAutoLottos(int lottoCount) {
	        return Stream.generate(LottoAutoGenerator::getAutoNumbers)
	                .limit(lottoCount);
	}

	public static Lottos mergeLottos(String lottoNumbers, int lottoCount) {
		return Stream.of(
		            purchaseManualLottos(lottoNumbers),
   		            purchaseAutoLottos(lottoCount))
		        .flatMap(Collection::stream)
		        .collect(Collectors.collectingAndThen(Collectors.toList(), Lottos::new));
	}
```
이렇게 하면 더 직관적으로 표현할 수 있습니다!